lms/djangoapps/commerce/views.py

```diff
@@ -10,6 +10,7 @@
 from lms.djangoapps.verify_student.models import SoftwareSecurePhotoVerification
 from shoppingcart.processors.CyberSource2 import is_user_payment_error
 from django.utils.translation import ugettext as _
+from openedx.core.djangoapps.theming import helpers
```

Comment:
`from openedx.core.djangoapps.theming import helpers as theming_helpers`

OR

`from openedx.core.djangoapps.theming.helpers import is_request_in_themed_site`

with the associated change when calling this function below.